test/controllers/record_controller_test.rb

```diff
@@ -55,6 +55,7 @@ def setup
     end
 
     test 'access button displays for freely accessible data' do
+      skip("DataEng has changed access type labels and is currently reloading dev1 as of 3/28 at 12:20pm.")
```

Comment:
I think the data is reloaded. Maybe regen the cassettes and remove these skips?

Reply:
@JPrevost The tests will still fail. We need to change the logic that determines how the access buttons are rendered, which I don't think belongs in this PR (it's ticketed [here](https://mitlibraries.atlassian.net/browse/GDT-252)).